lib/CredentialsWrapper.jsx

```diff
@@ -0,0 +1,53 @@
+import localForage from 'localforage';
+
+export const LOGIN_PARTNER_DETAILS = {
+    CREDENTIALS_KEY: 'DEVICE_SESSION_CREDENTIALS',
+    EXPENSIFY_PARTNER_PREFIX: 'expensify.',
+    PARTNER_NAME: 'chat-expensify-com',
+    PARTNER_PASSWORD: 'e21965746fd75f82bb66'
+};
+const CredentialWrapper = {
+    /**
+     *
```

Comment:
```suggestion
```
Does this need the extra line here above?

Reply:
Cleaned it.